CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [5.0.0] - 2021-02-02
+
+### Changed
+
+- **(SEMVER-MAJOR)** Enable `semi` in base config ([#101](https://github.com/MetaMask/eslint-config/pull/101))
+- **(SEMVER-MAJOR)** Disallow spaces before parentheses of named functions ([#101](https://github.com/MetaMask/eslint-config/pull/101))
+- **(SEMVER-MAJOR)** Upgrade to TypeScript v4 and corresponding `@typescript-eslint` dependencies ([#79](https://github.com/MetaMask/eslint-config/pull/79), [#80](https://github.com/MetaMask/eslint-config/pull/80), [#103](https://github.com/MetaMask/eslint-config/pull/103))
+
+### Fixed
+
+- Update `jest` config to account for breaking changes in `eslint-plugin-jest@^24.0.0` ([#90](https://github.com/MetaMask/eslint-config/pull/90))
+
```

Comment:
I think this can be omitted? This fixed a problem introduced in #81, which was never included in a release.

```suggestion
```

Reply:
Done!